protocols/grpc/hyperstart.proto

```diff
@@ -42,10 +44,17 @@ message CreateContainerRequest {
 	Spec OCI = 4;
 }
 
-message StartContainerRequest {
+message ContainerRequest {
 	string container_id = 1;
 }
 
+message RemoveContainerRequest {
```

Comment:
Did you intend to type StopContainerRequest, according to the commit log?

Reply:
@bergwolf Yes, sorry.